cppa_slack_transcript_tracker/utils/slack_listener.py

```
@@ -0,0 +1,222 @@
+            logger.debug("File shared event received")
+
+        @self.app.event("reaction_added")
+        def handle_reaction_added(event, body):
+            logger.debug("Reaction added event received")
+
+        @self.app.event("app_mention")
+        def handle_app_mention(event, body):
+            logger.debug("App mention event received")
+
+        @self.app.event({"type": "event_callback"})
+        def handle_all_events(event, body):
```

Ruff (0.15.2)

[warning] Unused function argument: `event`

(ARG001)